src/llmcompressor/transformers/finetune/trainer.py

```diff
@@ -6,4 +7,27 @@
 
 
 class Trainer(SessionManagerMixIn, HFTransformersTrainer):
-    pass
+    def __init__(self, **kwargs):
```

Comment:
why not in SparseAutoModel?

Reply:
anything downstream of HFTransformersTrainer init should be ok. HFTransformersTrainer init generates the output folder

Reply:
Lemme check what i can do w SparseAutoModel

Reply:
SparseAutoModel gets called before HFTransformersTrainer, so there wont be a place to save the .py files.